azure-pipelines.yml

```diff
@@ -0,0 +1,35 @@
+jobs:
+- job: lint
+  displayName: 'Lint'
+
+  pool:
+    vmImage: 'Ubuntu 16.04'
+
+  steps:
+  - script: sudo apt-get install python-setuptools
+    displayName: 'Install base dependencies'
+
+  - task: UsePythonVersion@0
+    displayName: 'Set Python version'
+    inputs:
+      versionSpec: '3.7.x'
+      addToPath: true
+
+  - script: pip install -r requirements-ci.txt
+    displayName: 'Install project using pip'
+
+  - script: python -m flake8 .
+    displayName: 'Run linter'
+
+- job: build
+  displayName: 'Build'
+  
+  pool:
+    vmImage: 'Ubuntu 16.04'
+
+  steps:
+  - script: sudo apt-get install docker
+    displayName: 'Install base dependencies'
+
+  - script: docker build -t cd-bot .
+    displayName: 'Build docker container'
```

Comment:
We need a `deploy` step to run `deploy.sh`.

Reply:
@AnonGuy Same deploy.sh as before?

Reply:
Deploy may be changing, please hold for updates, but it won't be a bash script, I can tell you that.

Reply:
smh I just reverted the commit for it